src/cdk/bootstrap.ts

```diff
@@ -339,6 +346,17 @@ export class MiraBootstrap {
         }
         break
       case 'cicd':
+        if (!this.args.file) {
```

Comment:
cicd does not accept a file parameter afaik, from where this requirement comes from?

Reply:
Good catch, I did by mistake, just fixed.